frontend/src/metabase/dashboard/containers/DashboardApp/DashboardApp.tsx

```diff
@@ -249,6 +197,9 @@ const DashboardApp = (props: DashboardAppProps) => {
     <div className={cx(CS.shrinkBelowContentSize, CS.fullHeight)}>
       <LeaveConfirmationModal isEnabled={isEditing && isDirty} route={route} />
 
+      {/* Suppressing for now until we can get the prop-drilled types sorted out. Previously DashboardControls was a JS file so types weren't checked, but now there's a Pandora's box here */}
```

Comment:
The fact that `DashboardControls` is a typescript file is causing a lot of issues with these older components. I can fix this in a separate PR